pyp2rpm/metadata_extractors.py

```diff
@@ -38,6 +103,72 @@ def name_convert_deps_list(self, deps_list):
         return deps_list
 
     @property
+    def has_pth(self):
+        """Figure out if package has pth file """
+        return "." in self.name
+
+    @property
+    def has_extension(self):
+        """Finds out whether the packages has binary extension.
+        Returns:
+            True if the package has a binary extension, False otherwise
+        """
+        return self.archive.has_file_with_suffix(settings.EXTENSION_SUFFIXES)
+
+
+    @property
+    def data_from_venv(self):
+        """Returns all metadata extractable from virtualenv object.
+        Returns:
+            dictionary containing metadata extracted from virtualenv
+        """
+        if not self.venv:
+            return {}
+
+        temp_dir = tempfile.mkdtemp()
+        try:
+            extractor = virtualenv.VirtualEnv(self.name, temp_dir,
+                                              self.name_convertor, 
+                                              self.base_python_version)
+            return extractor.get_venv_data
+        except VirtualenvFailException:
+             logger.error("Skipping virtualenv metadata extraction")
```

Comment:
I would add "...because of error"

It would be also very kind for the user to see what error happened.